arcgis-ios-sdk-samples/Geometry/Convex hull/README.md

```diff
@@ -0,0 +1,27 @@
+# Convex hull
+
+Create a convex hull for a given set of points. The convex hull is a polygon with shortest perimeter that encloses a set of points. As a visual analogy, consider a set of points as nails in a board. The convex hull of the points would be like a rubber band stretched around the outermost nails.
+
+![Create a convex hull](image1.png)
```

Comment:
Should we leave this image not renamed? This sample was created before the README updates started.